pom.xml

```diff
@@ -47,8 +47,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.3</version>
                 <configuration>
-                    <source>1.6</source>
-                    <target>1.6</target>
+                    <source>12</source>
+                    <target>12</target>
```

Comment:
Think it had to stay on 1.6 because that's what pdf2dom targets (but I forget how java works it's been a minute since I've been in jvm land)

Reply:
Ok, I understad!